autrainer/core/utils/bookkeeping.py

```diff
@@ -101,6 +101,76 @@ def create_folder(self, folder_name: str, path: str = "") -> None:
             exist_ok=True,
         )
 
+    def save_dataset_summary(
+        self,
+        data: "AbstractDataset",
+        filename: str,
+        val_exists: bool = False,
```

Comment:
How do we get this? I think `val_exists=True` is unreachable from the current setup in the `training.py`.

Reply:
Yes, I know. I have it here as an option, but we would need some rule on how to determine whether to do it for validation or not. I think the 2 possibilities where we do not want it is if 1) no validation exists (obviously) and 2) if validation is equal to test. The latter I am not sure how to determine easily, do you have an idea @ramppdev?

Reply:
At the moment, a validation dataset always exists, but could be equal to the test dataset. So the easiest way is to just always output it.
If we choose to move the logic to the datasets we could individually handle it there.

Reply:
With the new format #54, it should be more or less always there for each dataset (even if it is identical)